Pod/Classes/SEGAdjustIntegration.h

```diff
@@ -1,7 +1,12 @@
 #import <Foundation/Foundation.h>
-#import <Analytics/SEGAnalytics.h>
 #import <Adjust/Adjust.h>
 
+#if defined(__has_include) && __has_include(<Analytics/SEGAnalytics.h>)
+#import <Analytics/SEGAnalytics.h>
```

Comment:
Not quite sure if this header is correct. Seems odd/incorrect that we would check to see if `Analytics/SEGAnalytics.h` is defined before attempting to import it.

Reply:
Per this: https://clang.llvm.org/docs/LanguageExtensions.html#include-file-checking-macros

> The __has_include and __has_include_next macros allow you to check for the existence of an include file before doing a possibly failing #include directive.

#include synonymous with #import for all intensive purposes. So as long as the customer ran `pod install` this should check which version of Analytics they are using (ie. old namespace vs. new)

Reply:
Thank you!!!